README.md

```diff
@@ -15,6 +15,12 @@ pip install wavefront_lambda
 WAVEFRONT_URL = https://\<INSTANCE>.wavefront.com  
 WAVEFRONT_API_TOKEN = Wavefront API token with Direct Data Ingestion permission.  
 REPORT_STANDARD_METRICS = Set to False or false to not report standard lambda metrics directly to wavefront.  
+WAVEFRONT_PREFIX = Wavefront prefix. default set to empty
+
+WAVEFRONT_PROXY = If this environment exist and set to any non-zero value. Will use wavefront proxy. (WAVEFRONT_URL need to set to wavefront proxy hostname)
```

Comment:
Also mention, WAVEFRONT_API_TOKEN is not required in case proxy is being used.

Reply:
README.md updated